mlflow/genai/scorers/builtin_scorers.py

```diff
@@ -27,6 +27,38 @@ class _ChunkRelevance(_BaseBuiltInScorer):
 def chunk_relevance():
     """
     Chunk relevance measures whether each chunk is relevant to the input request.
+
+    Example (direct usage):
+    .. code-block:: python
+
+        import mlflow
+        from mlflow.genai.scorers import chunk_relevance
+
+        assessment = chunk_relevance()(
+            inputs="What is the capital of France?",
```

Comment:
```suggestion
            inputs={"question": "What is the capital of France?"},
```

Sorry for the last minutes update - there was a product decision that we will only support dictionary input. Can we update examples to follow that and validate if they work?